### DISCUSSION TITLE
Format for submission

### DISCUSSION THREAD

**Discussion by abaumane** [IBAHCM/RPiR#186] [Archive] [answered]
Hi @richardreeve I just wanted to double-check how we should submit the practicals.

If all you need us to submit are links, can we just use the "Online text" window on the submission portal to copy in the links to our repos or does a submission have to include a file of some sort? Sorry if it's a silly question.

**Comment by richardreeve** [comment, marked as answer]
Yes, online text is fine. If you prefer you can upload a plain text file or a word document with the links in - either is fine.
